src/common/login.ts

```diff
@@ -34,9 +36,10 @@ const Login = (() => {
             : `${oauth_url}/oauth2/authorize?app_id=${deriv_app_id}&l=${language}&brand=${brand_name.toLowerCase()}${affiliate_token_link}${cookies_link}&platform=${sub_url}`
     }
 
-    type TSocialProvider = 'google' | 'facebook' | 'apple'
-    const initOneAll = (provider: TSocialProvider): void => {
-        const social_login_url = `${loginUrl()}&social_signup=${provider}`
+    const initOneAll = (provider: TSocialProvider, utm_content?: string): void => {
```

Comment:
in this section we didn't have `utm_content`.
where does that come from?

Reply:
It comes from `_get-book` component.

Reply:
We are using this function inside `_get-book` component!